scripts/cluster_merge.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+set -euo pipefail
+
+#
+# This script orchestrates the merging of multiple DV node configurations
+# from a source cluster into a destination cluster. It calls node_merge.sh
+# for each corresponding nodeX subfolder found.
+# For single node merges, consider using node_merge.sh directly.
+#
+# Please make backups before running this script.
+# Please read the README.md instructions carefully.
+#
+
+# --- Function to display script usage ---
+function usage() {
+    echo "Usage: $0 dst_cluster_folder src_cluster_folder"
+    echo ""
+    echo "  dst_cluster_folder: Path to the destination cluster folder"
+    echo "                      (e.g., ~/dst_cluster)"
+    echo "  src_cluster_folder: Path to the source cluster folder"
+    echo "                      (e.g., ~/src_cluster)"
+    exit 1
+}
+
+# --- Main script starts here ---
+
+# Check for correct number of arguments
+if [ "$#" -ne 2 ]; then
+    usage
+fi
+
+DST_CLUSTER_ROOT="$1"
+SRC_CLUSTER_ROOT="$2"
+
+# Get the directory where this script is located
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+NODE_MERGE_SCRIPT="$SCRIPT_DIR/node_merge.sh"
+
+# Check if node_merge.sh exists and is executable
+if [ ! -f "$NODE_MERGE_SCRIPT" ]; then
+    echo "Error: '$NODE_MERGE_SCRIPT' not found. Please ensure 'node_merge.sh' is in the same directory as 'cluster_merge.sh'."
+    exit 1
+fi
+if [ ! -x "$NODE_MERGE_SCRIPT" ]; then
+    echo "Error: '$NODE_MERGE_SCRIPT' is not executable. Please run 'chmod +x $NODE_MERGE_SCRIPT'."
+    exit 1
+fi
+
+echo "--- Starting cluster merge from '$SRC_CLUSTER_ROOT' to '$DST_CLUSTER_ROOT' ---"
+echo ""
+
+# Find all nodeX folders in the source cluster
```

Comment:
Shouldn't we do the same for destination folder?
Also ensure the same amount of nodes are found?

Reply:
None of these were done for greater simplification and the expected short lifecycle of the script (until we add `addvalidators` command probably).
Checking the same nodes is barely useful, because vast majority of clusters is either 4 or 7 nodes.. All this script is insecure and must be used by advanced users only.
If you want me to add some specific (critical?) checks, let's do that.